Improvements to the Step by step tutorial (#6622)
Particular/docs.particular.net: tutorials/nservicebus-step-by-step/2-sending-a-command/tutorial.md

A [**message**](/nservicebus/messaging/messages-events-commands.md) is a collection of data sent via one-way communication between two endpoints. In NServiceBus, we define messages via simple classes.

> In this lesson, we'll focus on a specific type of message: [commands](/nservicebus/messaging/messages-events-commands.md). Later on, in [Part 4: Publishing events](../4-publishing-events/) we'll expand to look at another type of message, events.

Reviewer: ```suggestion
In this lesson, we'll focus on a specific type of message: [commands](/nservicebus/messaging/messages-events-commands.md). Later on, in [Part 4: Publishing events](../4-publishing-events/) we'll expand to look at another type of message, `events`.
```

To define a command, create a class and mark it with the `ICommand` interface.

snippet: Command

> By implementing this interface we let NServiceBus know that the class is a command so that it can build up some metadata about the message type when the endpoint starts up. Any properties you create within the message will constitute the message data.

Reviewer: ```suggestion
By implementing this interface, we let NServiceBus know that the class is a command so that it can build up some metadata about the message type when the endpoint starts up. Any properties you create within the message constitutes the message data.
```

> The name of your command classes it is important, as they allow to infer the intent of the class without looking at its content. A command is an order to do something, so it should be named in the [imperative tense](https://en.wikipedia.org/wiki/Imperative_mood). 

Reviewer: ```suggestion
The naming of your command classes is important, as a well-named class suggests its purpose without requiring inspection of its contents. A command is an order to do something, so it should be named in the [imperative tense](https://en.wikipedia.org/wiki/Imperative_mood). 
```

> `PlaceOrder` and `ChargeCreditCard` are good names for commands, because they are phrased as a command and are very specific. We could expect that `PlaceOrder` will place an order and `ChargeCreditCard` will charge money on a credit card. `CustomerMessage`, on the other hand, is not a good example. It is not in the imperative, and it's vague. Ideally another person should know exactly what a command's purpose is just by reading its name.

Reviewer: ```suggestion
`PlaceOrder` and `ChargeCreditCard` are good names for commands, because they are phrased as commands and are very specific. We could expect that `PlaceOrder` will place an order and `ChargeCreditCard` will charge money on a credit card. `CustomerMessage`, on the other hand, is not a good name. It is not in the imperative, and it is vague. Ideally, another person should know exactly what a command's purpose is just by reading its name.
```

> Command names should also convey business intent. `UpdateCustomerPropertyXYZ`, while more specific than `CustomerMessage` isn't a good name for an command because it's focused only on the data manipulation rather than the business meaning behind it. `MarkCustomerAsGold`, or something else that is domain oriented, is a better choice.

Reviewer: ```suggestion
Command names should also convey business intent. For example, `UpdateCustomerPropertyXYZ`, while more specific than `CustomerMessage` isn't a good name for a command because it's focused only on the data manipulation rather than the business meaning behind it. `MarkCustomerAsGold`, or something else that is domain-oriented, is a better choice.
```
